native/src/main/java/io/xlibb/pipe/Pipe.java

```diff
@@ -115,7 +116,7 @@ public static BError produce(Environment env, BObject pipe, Object event, BDecim
             Pipe nativePipe = (Pipe) handle.getValue();
```

Comment:
Um bit confused here, why are we using a Ballerina `handle` at the native level to retrieve a native object ?